packages/product/src/migrations/Migration20230908084537.ts

```diff
@@ -0,0 +1,69 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20230908084537 extends Migration {
```

Comment:
Do we need to add the Feature Flag here and on all the Models?

Reply:
Not on those migrations, they were just forgotten during the implementation IMO